ci/federalist-pipeline.yml

```diff
@@ -0,0 +1,235 @@
+---
+############################
+#  SHARED
+
+env-cf: &env-cf
+  CF_API: https://api.fr.cloud.gov
+  CF_USERNAME: ((production-cf-username))
+  CF_PASSWORD: ((production-cf-password))
+  CF_ORG: gsa-18f-federalist
+  CF_SPACE: production
+
+python-image: &python-image
+  platform: linux
+  image_resource:
+    type: docker-image
+    source:
+      repository: python
+      tag: 3.8
+
+cf-image: &cf-image
+  platform: linux
+  image_resource:
+    type: docker-image
+    source:
+      repository: 18fgsa/concourse-task
+
+test: &test
+  - task: test
+    config:
+      <<: *python-image
+      inputs: [name: src]
+      outputs: [name: src]
+      run:
+        dir: src
+        path: ci/tasks/test.sh
+    params:
+      CC_TEST_REPORTER_ID: 449ef357d9e81688685bde0dcdd348bc764d717fa6a4c9c01562bd8037ceb665
+
+############################
+#  JOBS
+
+jobs:
+
+  - name: set-pipeline
+    plan:
+    - get: src
+      resource: src-production
+      trigger: true
+    - set_pipeline: federalist-build-container-production
+      file: src/ci/federalist-pipeline.yml
+
+  - name: test-pr-main
+    plan:
+    - get: src
+      resource: pr-main
+      trigger: true
+      version: every
+
+    - put: src
+      resource: pr-main
+      params:
+        path: src
+        status: pending
+        base_context: concourse
+        context: test
+
+    - do: *test
+
+    on_failure:
+      put: src
+      resource: pr-main
+      params:
+        path: src
+        status: failure
+        base_context: concourse
+        context: test
+
+    on_success:
+      put: src
+      resource: pr-main
+      params:
+        path: src
+        status: success
+        base_context: concourse
+        context: test
```

Comment:
Update `context` to something like `test-federalist` so we don't conflict with pages production PR check.